### ISSUE TITLE
Configuration is not Working for Show Related videos in Product

### ISSUE THREAD

**Issue by mageprashant** [magento/magento2#19415]
**Preconditions**

1. Magento 2.2.6

**Steps to reproduce**

1. Go to Magento Backend (Stores -> Configuration -> Catalog -> Catalog -> Product Video)
          > Set YouTube API Key
          > Make Sure Show related video is No [see more](http://nimb.ws/TuLs2X)
2. Upload Product Video in Product (Catalog -> Product -> Open Any Prodcut For Edit -> Images And Videos -> Add Video) [see more](https://www.screencast.com/t/z14awXjv)
         > Add YouTube Video Url in Url [see more](https://www.screencast.com/t/791xCAswycRv)

**Expected result**

- Related videos are not displayed

**Actual result**

- Related videos are displayed Frontend in PDP [see more](https://www.screencast.com/t/zQ4sW5hwEYcQ)

**Comment by magento-engcom-team** [comment]
Hi @mageprashant. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@mageprashant do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by mageprashant** [comment]
@magento-engcom-team give me 2.2.6 instance

**Comment by magento-engcom-team** [comment]
Hi @mageprashant. Thank you for your request. I'm working on Magento 2.2.6 instance for you

**Comment by magento-engcom-team** [comment]
Hi @mageprashant, here is your [Magento instance](https://i-19415-2-2-6.instances.magento-community.engineering).
Admin access: [https://i-19415-2-2-6.instances.magento-community.engineering/admin](https://i-19415-2-2-6.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by mageprashant** [comment]
Hi @magento-engcom-team I confirm I can reproduce the issue on your vanilla instance.	

**Steps to reproduce**

1. Go to Magento Backend (Stores -> Configuration -> Catalog -> Catalog -> Product Video)
          > Set YouTube API Key
          > Make Sure Show related video is No
2. Upload Product Video in Product (Catalog -> Product -> Open Any Prodcut For Edit -> Images And Videos -> Add Video)\
         > Add YouTube Video Url in Url

**Expected result**

- Related videos are not displayed

**Actual result**

- Related videos are displayed Frontend in PDP > https://i-19415-2-2-6.instances.magento-community.engineering/related-videos.html

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by gelanivishal** [comment]
Hello @mageprashant, Please check https://i-19415-2-2-6.instances.magento-community.engineering/catalog/product/view/id/1/ I have also added new video, You can see related product video link on it.

Let me know once you have checked.

**Comment by mageprashant** [comment]
Hi @gelanivishal 
Please can you check proper Steps to reproduce
I want to disable related video for youtube in Magento PDP

**Comment by magento-engcom-team** [comment]
@engcom-backlog-nazar Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-96652`, `MAGETWO-96653` were created

**Comment by magento-engcom-team** [comment]
Hi @Dharmeshvaja91. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by Dharmeshvaja91** [comment]
@magento-engcom-team give me 2.2.6 instance

**Comment by magento-engcom-team** [comment]
Hi @Dharmeshvaja91. Thank you for your request. I'm working on Magento 2.2.6 instance for you

**Comment by magento-engcom-team** [comment]
Hi @Dharmeshvaja91, here is your [Magento instance](https://i-19415-2-2-6.instances.magento-community.engineering).
Admin access: [https://i-19415-2-2-6.instances.magento-community.engineering/admin](https://i-19415-2-2-6.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by Dharmeshvaja91** [comment]
Hello, @mageprashant @magento-engcom-team i am not able to reproduce this issue on 2.2.6 instance.

**Comment by magento-engcom-team** [comment]
@mageprashant, thank you for your report.     
    
Unfortunately, we are archiving this ticket now as it did not get much attention from both Magento Community and Core developers for an extended period. This is done in an effort to create a quality, community-driven backlog which will allow us to allocate the required attention more easily.

Please feel free to comment, reopen or create new ticket according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) 
if you are still facing this issue on the latest `2.3-develop` branch. Thank you for collaboration.

**Comment by sb-nick** [comment]
Hi,
I'm running 2.3.3-p1 and this setting "Show Related videos" = N isn't preventing other videos from loading. I see that Magento closed this for no attention, but fact remains there is still a bug here to be fixed. I really don't want videos not related to my company to show up after a video is played.
